lib/render.js

```diff
@@ -10,6 +10,12 @@ const defaultMarkedRenderer = marked.defaults.renderer || new marked.Renderer()
 marked.use({
   renderer: {
     code (code, infostring, escaped) {
+      // Ignore regular code blocs, rendering the raw content on the page itself
```

Comment:
Typo?
```suggestion
      // Ignore regular code blocks, rendering the raw content on the page itself
```

Reply:
Similr thing on the following line. I'm actually not sure if it's supposed to be 'block' or 'bloc' but searching suggests the former. Not gonna die on that hill.